### PR TITLE
chore(store): translate CJK comments to English for store-cli, store-client, store-common, store-core

### PR DESCRIPTION
**Pull request by VGalaxies**
subtask of https://github.com/apache/incubator-hugegraph/issues/2575

### CODE REVIEW

File: hugegraph-store/hg-store-core/src/main/java/org/apache/hugegraph/store/PartitionEngine.java [hunk 1]
```diff
@@ -253,16 +253,16 @@ public LogStorage createLogStorage(final String uri, final RaftOptions raftOptio
...
                                                      serverId, nodeOptions,
                                                      storeEngine.getRaftRpcServer(), true);
         this.raftNode = raftGroupService.start(false);
         this.raftNode.addReplicatorStateListener(new ReplicatorStateListener());
 
-        // 检查pd返回的peers是否和本地一致，如果不一致，重置peerlist
+        // Check if the peers returned by pd are consistent with the local ones, if not, reset the peerlist
         if (this.raftNode != null) {
-            //TODO 检查peer列表，如果peer发生改变，进行重置
+            //TODO Check peer list, if peer changes, perform reset
```
Comment (VGalaxies) [inline hugegraph-store/hg-store-core/src/main/java/org/apache/hugegraph/store/PartitionEngine.java:265]:
```suggestion
            // TODO: Check peer list, if peer changes, perform reset
```

File: hugegraph-store/hg-store-core/src/main/java/org/apache/hugegraph/store/PartitionEngine.java [hunk 2]
```diff
@@ -398,19 +398,19 @@ public Status changePeers(List<String> peers, final Closure done) {
...
                 if (!removedPeers.isEmpty()) {
                     removedPeers.forEach(peer -> Utils.runInThread(() -> {
-                        // 6. 停止已被删除的peer
+                        // 6. Stop the deleted peer
                         rpcClient.destroyRaftNode(peer,
                                                   partitionManager.getPartitionList(getGroupId()),
                                                   status -> {
                                                       if (!status.isOk()) {
-                                                          // TODO 失败了怎么办？
+                                                          // TODO What if it fails?
```
Comment (VGalaxies) [inline hugegraph-store/hg-store-core/src/main/java/org/apache/hugegraph/store/PartitionEngine.java:413]:
```suggestion
                                                          // TODO: What if it fails?
```

File: hugegraph-store/hg-store-core/src/main/java/org/apache/hugegraph/store/PartitionEngine.java [hunk 3]
```diff
@@ -563,7 +563,7 @@ public Endpoint waitForLeader(long timeOut) {
                     if (partitionManager.isLocalPartition(this.options.getGroupId())) {
                         log.error("Raft {} leader not found, try to repair!",
                                   this.options.getGroupId());
-                        // TODO 判断raft是否本机，如果是，尝试修复Leader，包括检查配置是否正确
+                        // TODO Check if raft is local, if so, try to fix the Leader, including checking if the configuration is correct.
```
Comment (VGalaxies) [inline hugegraph-store/hg-store-core/src/main/java/org/apache/hugegraph/store/PartitionEngine.java:566]:
```suggestion
                        // TODO: Check if raft is local, if so, try to fix the Leader, including checking if the configuration is correct.
```

File: hugegraph-store/hg-store-core/src/main/java/org/apache/hugegraph/store/PartitionEngine.java [hunk 4]
```diff
@@ -630,9 +630,9 @@ public void onStartFollowing(final PeerId newLeaderId, final long newTerm) {
     public void onConfigurationCommitted(Configuration conf) {
 
         try {
-            //更新shardlist
+            //Update shardlist
```
Comment (VGalaxies) [inline hugegraph-store/hg-store-core/src/main/java/org/apache/hugegraph/store/PartitionEngine.java:633]:
```suggestion
            // Update shardlist
```

File: hugegraph-store/hg-store-core/src/main/java/org/apache/hugegraph/store/PartitionEngine.java [hunk 5]
```diff
@@ -965,29 +965,29 @@ private Status handleMoveTask(MetaTask.Task task) {
...
+        // key range validation
         if (partition != null && partition.getEndKey() == keyEnd &&
             partition.getStartKey() == keyStart) {
             log.info("remove partition id :{}, graph:{}", partition.getId(),
                      partition.getGraphName());
             storeEngine.deletePartition(partitionId, graphName);
         }
 
-        // 没有partition engine的情况
+        // without partition engine的情况
```
Comment (VGalaxies) [inline hugegraph-store/hg-store-core/src/main/java/org/apache/hugegraph/store/PartitionEngine.java:990]:
```suggestion
        // No partition engine present
```

File: hugegraph-store/hg-store-core/src/main/java/org/apache/hugegraph/store/business/BusinessHandler.java [hunk 6]
```diff
@@ -108,23 +108,23 @@ void loadSnapshot(String snapshotPath, String graph, int partId, long version) t
...
-    // 扫描分区从 seqnum 开始的 kv
+    // Scan partition starting from seqnum for kv
     ScanIterator scanRaw(String graph, int partId, long seqNum) throws HgStoreException;
 
     void ingestSstFile(String graph, int partId, Map<byte[], List<String>> sstFiles) throws
                                                                                      HgStoreException;
 
-    //提交分区分裂，删除旧数据
-    // 删除分区数据
+    //Submit partition split, delete old data
```
Comment (VGalaxies) [inline hugegraph-store/hg-store-core/src/main/java/org/apache/hugegraph/store/business/BusinessHandler.java:117]:
```suggestion
    // Submit partition split, delete old data
```

File: hugegraph-store/hg-store-core/src/main/java/org/apache/hugegraph/store/business/BusinessHandler.java [hunk 7]
```diff
@@ -108,23 +108,23 @@ void loadSnapshot(String snapshotPath, String graph, int partId, long version) t
...
+    // Delete partition data
     boolean deletePartition(String graph, int partId);
 
-    //清理分区，删除多余的数据
+    //Clean up partitions, delete unnecessary data
```
Comment (VGalaxies) [inline hugegraph-store/hg-store-core/src/main/java/org/apache/hugegraph/store/business/BusinessHandler.java:121]:
```suggestion
    // Clean up partitions, delete unnecessary data
```

File: hugegraph-store/hg-store-core/src/main/java/org/apache/hugegraph/store/business/BusinessHandler.java [hunk 8]
```diff
@@ -108,23 +108,23 @@ void loadSnapshot(String snapshotPath, String graph, int partId, long version) t
...
     boolean cleanPartition(String graph, int partId);
 
     boolean cleanPartition(String graph, int partId, long startKey, long endKey,
                            CleanType cleanType);
 
-    //所有指定分区图的所有 table 名
+    // All table names for all specified partition diagrams
```
Comment (VGalaxies) [inline hugegraph-store/hg-store-core/src/main/java/org/apache/hugegraph/store/business/BusinessHandler.java:127]:
```suggestion
    // All table names for all specified partition graph
```

File: hugegraph-store/hg-store-core/src/main/java/org/apache/hugegraph/store/business/BusinessHandlerImpl.java [hunk 9]
```diff
@@ -719,7 +719,7 @@ public void createTable(String graph, int partId, String table) {
     @Override
     public void deleteTable(String graph, int partId, String table) {
         dropTable(graph, partId, table);
-        // todo 检查表是否为空，为空则真实删除表
+        // todo Check if the table is empty, if empty then truly delete the table
```
Comment (VGalaxies) [inline hugegraph-store/hg-store-core/src/main/java/org/apache/hugegraph/store/business/BusinessHandlerImpl.java:722]:
```suggestion
        // TODO: Check if the table is empty, if empty then truly delete the table
```

File: hugegraph-store/hg-store-core/src/main/java/org/apache/hugegraph/store/meta/StoreMetadata.java [hunk 10]
```diff
@@ -197,12 +197,12 @@ public String getPartitionStoreLocation(int partitionId, String dbName) {
...
                 location = getPartitionStore(partitionId);
                 if (location == null) {
-                    // 查找分区数最少的存储
+                    // Find the storage with the least number of partitions
                     location = Metapb.PartitionStore.newBuilder()
                                                     .setPartitionId(partitionId)
                                                     .setStoreLocation(getMinDataLocation())
                                                     .build();
-                    // TODO 选择分区数最小的路径
+                    // TODO Select the path with the least number of partitions.
```
Comment (VGalaxies) [inline hugegraph-store/hg-store-core/src/main/java/org/apache/hugegraph/store/meta/StoreMetadata.java:205]:
```suggestion
                    // TODO: Select the path with the least number of partitions.
```

**Review by VGalaxies — COMMENTED** [review]
suggestions batch #1

File: hugegraph-store/hg-store-core/src/main/java/org/apache/hugegraph/store/meta/StoreMetadata.java [hunk 11]
```diff
@@ -216,12 +216,12 @@ public String getPartitionRaftLocation(int partitionId) {
...
                 location = getPartitionRaft(partitionId);
                 if (location == null) {
-                    // 查找分区数最少的存储
+                    // Find the storage with the least number of partitions
                     location = Metapb.PartitionRaft.newBuilder()
                                                    .setPartitionId(partitionId)
                                                    .setRaftLocation(getMinRaftLocation())
                                                    .build();
-                    // TODO 选择分区数最小的路径
+                    // TODO Select the path with the fewest partitions.
```
Comment (VGalaxies) [inline hugegraph-store/hg-store-core/src/main/java/org/apache/hugegraph/store/meta/StoreMetadata.java:224]:
```suggestion
                    // TODO: Select the path with the fewest partitions.
```

File: hugegraph-store/hg-store-core/src/main/java/org/apache/hugegraph/store/raft/RaftOperation.java [hunk 12]
```diff
@@ -87,7 +87,7 @@ public static RaftOperation create(final byte op, final Object req) {
     public static RaftOperation create(final byte op,
                                        final com.google.protobuf.GeneratedMessageV3 req) throws
                                                                                          IOException {
-        // 序列化，
+        // Serialization,
```
Comment (VGalaxies) [inline hugegraph-store/hg-store-core/src/main/java/org/apache/hugegraph/store/raft/RaftOperation.java:90]:
```suggestion
        // Serialization
```

File: hugegraph-store/hg-store-core/src/main/java/org/apache/hugegraph/store/util/HgStoreException.java [hunk 13]
```diff
@@ -21,7 +21,7 @@ public class HgStoreException extends RuntimeException {
 
     public final static int EC_NOEXCEPT = 0;
     public final static int EC_FAIL = 1000;
-    //存储的数据格式不支持
+    //The data format stored is not supported.
```
Comment (VGalaxies) [inline hugegraph-store/hg-store-core/src/main/java/org/apache/hugegraph/store/util/HgStoreException.java:24]:
```suggestion
    // The data format stored is not supported.
```

**Review by VGalaxies — COMMENTED** [review]
suggestions batch 2

**Review by Pengzna — APPROVED** [review]
LGTM!!

LLM yes, Go**gle translator s**t

image

**Review by imbajin — APPROVED** [review]

**Review by JackyYangPassion — APPROVED** [review]

**Pull request merged**